Docker/Runner/DataDirectory.php

```diff
@@ -42,6 +42,7 @@ public function createDataDir()
         $fs->mkdir($this->workingDir);
```

Comment:
hm, asi by se teda sluselo prejmenovat DataDirectory na WorkingDirectory, protoze to je co to dela https://github.com/keboola/docker-bundle/issues/277

Reply:
Bingo. Chtěl jsem nejdřív udělat klon `DataDirectory` => `TempDirectory`, ale to by byl hroznej vomrd, takže `WorkingDirectory` je lepší.

Reply:
Jsi mě předběhl s mergem :-)

Reply:
Udělám to rovnou jako další PR

Reply:
vsak jsem na to udelal issue, abysme tohle nezdrzovali